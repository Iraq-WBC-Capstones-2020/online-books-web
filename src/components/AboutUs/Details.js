import React from 'react';
import pic from './media/pic.png';
import { MDBRow, MDBCol, MDBContainer, MDBBtn } from 'mdbreact';
import { useTranslation } from 'react-i18next';
import Paragraph from '../../components/Paragraph/Paragraph';

function Details() {
  const { t } = useTranslation();
  return (
    <>
      <MDBContainer>
        <MDBRow>
          <MDBCol lg="5">
            <img src={pic} alt="artwork of a girl reading a book" />
          </MDBCol>
          <MDBCol>
            <div className="mt-24">
              <Paragraph title={t('aboutPage.aboutSection.aboutUs')}>
                {t('aboutPage.aboutSection.aboutUsText')}
              </Paragraph>
              <MDBBtn color="primary" className="rounded-pill " size="md">
                {t('aboutPage.aboutSection.readMoreButton')}
              </MDBBtn>
            </div>
          </MDBCol>
        </MDBRow>
      </MDBContainer>
    </>
  );
}

export default Details;
