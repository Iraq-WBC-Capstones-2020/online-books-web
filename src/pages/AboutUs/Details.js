import React from 'react';
import pic from './pic.png';
import { MDBRow, MDBCol } from 'mdbreact';
import { useTranslation } from 'react-i18next';

function Details() {
  const { t } = useTranslation();

  const aboutPageTranslation = {
    headerTitle: t('aboutPage.aboutSection.aboutUs'),
    text: t('aboutPage.aboutSection.aboutUsText'),
    ReadMore: t('aboutPage.aboutSection.ReadMoreButton'),
  };
  return (
    <div className="bg-gray-100 col">
      <MDBRow>
        <MDBCol lg="5">
          <img src={pic} alt="artwork of a girl reading a book" />
        </MDBCol>
        <MDBCol>
          <div className="mt-48">
            <h3 className="font-weight-bold text-3xl ml-1">
              {aboutPageTranslation.headerTitle} :
            </h3>
            <hr className="col-lg-2 h-2 bg-blue-500" />
            <p className="mt-2 mr-5 text-lg text-justify">
              {aboutPageTranslation.text}
            </p>
            <button className="rounded-full h-8 w-1/9 mt-3 transition duration-500 ease-in-out bg-blue-500 hover:bg-blue-700 transform hover:-translate-y-1 hover:scale-110">
              <span className="text-white">
                {aboutPageTranslation.ReadMore}
              </span>
            </button>
          </div>
        </MDBCol>
      </MDBRow>
    </div>
  );
}

export default Details;
