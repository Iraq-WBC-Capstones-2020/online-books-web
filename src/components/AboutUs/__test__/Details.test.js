import React from 'react';
import renderer from 'react-test-renderer';
import Details from './../Details';

test('Details snapshot', () => {
  const tree = renderer.create(<Details />).toJSON();
  expect(tree).toMatchSnapshot();
});
