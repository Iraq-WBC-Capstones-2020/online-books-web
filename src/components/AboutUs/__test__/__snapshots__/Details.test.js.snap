// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details snapshot 1`] = `
<div
  className="bg-gray-100"
>
  <div
    className="container"
    data-test="container"
  >
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-lg-5"
        data-test="col"
      >
        <img
          alt="artwork of a girl reading a book"
          src="pic.png"
        />
      </div>
      <div
        className="col"
        data-test="col"
      >
        <div
          className="mt-24"
        >
          <p
            className="text-3xl font-medium border-b-4 inline border-blue-500"
          >
            aboutPage.aboutSection.aboutUs
          </p>
          <p
            className="text-base leading-loose text-justify"
          >
            aboutPage.aboutSection.aboutUsText
          </p>
          <button
            className="btn-primary btn Ripple-parent btn-sm rounded-pill"
            data-test="button"
            onMouseUp={[Function]}
            onTouchStart={[Function]}
            type="button"
          >
            aboutPage.aboutSection.readMoreButton
            <div
              className="Ripple "
              data-test="waves"
              style={
                Object {
                  "height": "0px",
                  "left": "0px",
                  "top": "0px",
                  "width": "0px",
                }
              }
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
